src/axel_f/excel/coerce.cljc

```diff
@@ -27,3 +28,62 @@
     true "TRUE"
     false "FALSE"
     (str item)))
+
+(defn to-string*
+  "Tries to coerce given value to a string type. Returns null for empty value."
+  [^{:doc "Any object to coerce to a string."} obj]
+  (when obj (str obj)))
+
+(def to-string #'to-string*)
+
+(defn to-integer*
+  "Tries to coerce given value to an integer type. Returns null for empty or not reducible to an integer type value."
```

Comment:
For this and subsequent cases, I'd try to rephrase the second half. 

Maybe something like:

> Returns null for empty values or values not reducible to integer

Reply:
thanks!